grails-app/services/au/org/ala/merit/ProjectService.groovy

```diff
@@ -774,7 +774,10 @@ class ProjectService  {
             response = updateUnchecked(projectId, [plannedStartDate:plannedStartDate, plannedEndDate:plannedEndDate])
 
             //user explicitly generates the report from the reporting tab
-            generateProjectStageReports(projectId, dateChangeOptions)
+            if (!APPLICATION_STATUS.equalsIgnoreCase(project.status)) {
```

Comment:
You only needed to include this change because you also changed canGenerateReports to return true when the project is in application status so it can be removed when that is fixed.